lib/utils/timed-callback.js

```diff
@@ -51,6 +51,7 @@ class TimedCallback {
     this.timeoutId = setTimeout(() => {
       const err = new TimeoutError(`done() callback timeout of ${this.timeoutMs}ms was reached while executing "${this.name}".` +
         ' Make sure to call the done() callback when the operation finishes.');
+      err.help =['Setting the asyncHookTimeout globals config to the required value'];
```

Comment:
While how you've set the help property is correct, I don't think this is the correct place to add this since `timed-callback` might be used at other places as well and not just the hooks.

You might need to backtrack a little to reach a part of code which is only shared by hooks, and modify the err object there. VSCode Debugger is a great tool to walk through the codebase as it is running and backtrack the steps using the Call Stack feature of debugger.